project-1/main.py

```diff
@@ -0,0 +1,193 @@
+# pylint: disable=import-error
+from pathlib import Path
+from google.oauth2 import service_account
+from gspread_dataframe import set_with_dataframe
+from exceptions import *
```

Comment:
I'm anti "import *" - LOL, let's see all we're importing, it makes the code easier to read

Reply:
sorry for my habit.. LOL